projects/Mallard/src/components/layout/header/EditionMenuScreenHeader.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```

Comment:
We have a `ScreenHeader` folder inside components where this should live.

Reply:
@jimhunty do you want to have another look, please?